kafka_consumer/assets/configuration/spec.yaml

```diff
@@ -62,8 +62,28 @@ files:
       - name: consumer_groups
         description: |
           Each level is optional. Any empty values are fetched from the Kafka cluster.
-          You can have empty partitions (example: <CONSUMER_NAME_2>), topics (example: <CONSUMER_NAME_3>),
-          and even consumer_groups. If you omit consumer_groups, you must set `monitor_unlisted_consumer_groups` to true.
+          You can have empty partitions (example: <CONSUMER_NAME_2>), topics
+          (example: <CONSUMER_NAME_3>),
+          and even consumer_groups. If you do not specify `consumer_groups`,
+          you must set `monitor_unlisted_consumer_groups` to true. 
+          
+          If both `consumer_groups` and `monitor_unlisted_consumer_groups` are used,
+          then `consumer_groups` is ignored.
+        value:
+          type: object
+          example:
+            <CONSUMER_NAME_1>:
+              <TOPIC_NAME_1>: [0, 1, 4, 12]
+            <CONSUMER_NAME_2>:
+              <TOPIC_NAME_2>: []
+            <CONSUMER_NAME_3>: {}
```

Comment:
I think it would be helpful to provide some examples of proper regexes here - particularly to provide examples of keys like this in yaml (do they need to be wrapped in `"` or can you drop a raw regex as a key)?

Reply:
Hey @mwarkentin sorry just saw this. I've made a PR changing these examples to be more realistic: https://github.com/DataDog/integrations-core/pull/14533

Reply:
Thanks @yzhan289 !